engine/GameOrchestrator.js

```diff
@@ -346,11 +389,12 @@ define([
 				game: Object.assign({
 					seed: null,
 					teams,
+					startFrame: 0,
 				}, baseGameConfig),
 				play: Object.assign({
 					delay: 0,
 					speed: 0,
-					maxTime: 500,
+					checkbackTime: 1000,
```

Comment:
Just a UI feel. The 500-ms update interval feels a bit too much like the system is running choppily, while the 1000-ms update interval gives the feel of periodic updates without excessive waiting between updates. If you want, I can change that back.

Reply:
I see. The original low-ish value is so that users can still control things when it's running slowly (i.e. pressing stop can take effect within half a second of pressing it). Increasing that to 1 second is probably fine, although it reminds me that the new `incomplete` event won't give the user that control (if they try to step at a speed of 1000 in a game which takes 0.1 seconds per frame, but get bored 10 seconds in to the 100 second task, they'll still have to wait the extra 90 seconds before they can stop it, whereas previously it would stop within a second. I'll have a think about how that issue can be avoided).